drivers/hmis/app/graphql/types/hmis_schema/current_living_situation.rb

```diff
@@ -18,8 +18,8 @@ def self.configuration
     field :enrollment, HmisSchema::Enrollment, null: false
     field :user, HmisSchema::User, null: true
     field :client, HmisSchema::Client, null: false
-    hud_field :information_date
-    hud_field :current_living_situation, HmisSchema::Enums::Hud::CurrentLivingSituation
+    field :information_date, GraphQL::Types::ISO8601Date, null: true
```

Comment:
I assume we're still validating this so you can't add any new records with a blank date right?  Also, maybe we want to put a comment here that it differs from the spec because the database constraint isn't in place?

Reply:
Yes we still validate all that on input

Reply:
We validate it on the AR record (right now). So its not just happening on the form input side.